Lib/tokenize.py

```diff
@@ -243,8 +258,6 @@ def untokenize(self, iterable):
                     end_line, end_col = end
                     extra_chars = last_line.count("{{") + last_line.count("}}")
                     end = (end_line, end_col + extra_chars)
-            elif tok_type in (STRING, FSTRING_START) and self.prev_type in (STRING, FSTRING_END):
```

Comment:
Why are we removing this?

Reply:
It inserts an extra space before the backslash in cases like this:

```python
msg = "foo" \
    "bar"
```

Keeping that `elif`, this would untokenize to:
```diff
- msg = "foo" \
+ msg = "foo"  \
    "bar"
```

Since we now correctly count the amount of whitespace, it should not be needed anymore.

It also seems to insert an extra space for two consecutive string literals which is also not needed:

```python
'' ''
```